extensions/wikia/FacebookClient/scripts/preferences.js

```diff
@@ -91,6 +92,8 @@
 				controller: 'FacebookClient',
 				method: 'disconnectFromFB',
 				format: 'json',
+				data: {token: mw.user.tokens.get('editToken')},
+				type: 'POST',
```

Comment:
Yea, I saw that, but it's nice to not have to dip into the code to find out. Being explicit here is good.